src/python/pants/goal/run_tracker_test.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import datetime
+import time
+
+from freezegun import freeze_time
+
+from pants.base.exiter import PANTS_SUCCEEDED_EXIT_CODE
+from pants.goal.run_tracker import RunTracker
+from pants.testutil.option_util import create_options_bootstrapper
+
+
+@freeze_time(datetime.datetime(2020, 1, 1, 12, 0, 0), as_kwarg="frozen_time")
+def test_run_tracker_timing_output(**kwargs) -> None:
+    open("BUILDROOT", "w")
```

Comment:
this is probably a resource leak...

Reply:
Agreed. You can use `Path("BUILDROOT").touch()` instead.

Reply:
@eric @asherf it looks like there's a `PANTS_BUILDROOT_OVERRIDE` variable that is meant to create a fake buildroot for tests, cf. https://github.com/pantsbuild/pants/pull/11431/commits/f3410faf3f0c760c92ed9f13adf1454c2fc891b6